documentation/modules/rn-2.6.adoc

```diff
@@ -2,6 +2,277 @@
 //
 // * documentation/doc-Release_notes/master.adoc
 
-[id="rn-2.6_{context}"]
+[id="rn-26_{context}"]
 = {project-full} 2.6
 
+You can use {the-lc} {project-first} to migrate virtual machines from the following source providers to {virt} destination providers:
+
+* VMware vSphere
+* {rhv-full} ({rhv-short})
+* {osp}
+* Open Virtual Appliances (OVAs) that were created by VMware vSphere
+* Remote {virt} clusters
+
+The release notes describe technical changes, new features and enhancements, known issues, and resolved issues.
+
+[id="technical-changes-26_{context}"]
+== Technical changes
+
+This release has the following technical changes:
+
+
+.Simplified the creation of vSphere providers
+
+Previously, users had to specify a fingerprint when creating a vSphere provider. This is required to retrieve the fingerprint from the server that vCenter runs on. {project-short} no longer requires this fingerprint as an input but rather computes it from the specified certificate in case of secure connection, or automatically retrieves it from the server that runs vCenter/ESXi in case of insecure connection.
```

Comment:
```suggestion
In earlier releases of {project-short}, users had to specify a fingerprint when creating a vSphere provider. This is required to retrieve the fingerprint from the server that vCenter runs on. {project-short} no longer requires this fingerprint as an input but rather computes it from the specified certificate in case of secure connection, or automatically retrieves it from the server that runs vCenter/ESXi in case of insecure connection.
```